AUTHORS

```diff
@@ -3139,5 +3139,188 @@ Shigma <33423008+Shigma@users.noreply.github.com>
 atian25@qq.com <atian25@qq.com>
 Amila Welihinda <amilajack@gmail.com>
 schamberg97 <50446906+schamberg97@users.noreply.github.com>
+DrunkenPoney <mrquebec13@gmail.com>
+Christoph Tavan <dev@tavan.de>
+Clark Kozak <clarkkozak@protonmail.com>
+Michael Auderer <mike@silverstone.io>
+Linn Dahlgren <admin@opposite.biz>
+Shelley Vohr <codebytere@gmail.com>
+Ikko Ashimine <eltociear@gmail.com>
+Anatoly Korniltsev <korniltsev.anatoly@gmail.com>
+Danielle Adams <adamzdanielle@gmail.com>
```

Comment:
@danielleadams Looks like you've used two email addresses so I imagine we'll want to put one in `.mailmap`. Do you care which one is listed in AUTHORS?

Reply:
Just adamzdanielle is good here. I'm not using the other one for open source anymore, so just this one is good for now 👍🏼